File: Backend/Src/Core/Store.Domain/Categories/Models/Output/CategoryAttributeListOutput.cs
using Store.Domain.Attribute;
using Store.Service.Attributes;

namespace Store.Domain.Categories.Models.Output;

public class CategoryAttributeListOutput
{
    public int Id { get; set; }
    public int AttributeId { get; set; }
    public string? AttributeTitle { get; set; }
    public AttributeUnitEnum AttributeUnit { get; set; }
    public AttributeTypeEnum AttributeType { get; set; }
}

File: FrontendPanel/app/features/categories/models/output/category-attribute-list-output.ts
import type { AttributeType } from "~/features/attributes/models/enums/attribute-type";
import type { AttributeUnit } from "~/features/attributes/models/enums/attribute-unit";

export interface CategoryAttributeListOutput {
  id: number;
  categoryId: number;
  categoryTitle: string;
  attributeId: number;
  attributeTitle: string;
  attributeUnit: AttributeUnit;
  attributeType: AttributeType;
}
